src/AzureFunctions/Example/host.json

```diff
@@ -0,0 +1,3 @@
+{
+    "version": "2.0"
```

Comment:
What version is it? Is it related to target framework version?

Reply:
The Version of the host.json: https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json 
The file is required but no setting is needed.